src/backend/tests/work-packages.test.ts

```diff
@@ -91,4 +48,31 @@ describe('Work Packages', () => {
     expect(res.statusCode).toBe(400);
     expect(res.body.message).toBe('One of the dependencies was not found.');
   });
+  test('createWorkPackage fails if user does not have access', async () => {
+    
+    jest.spyOn(prisma.user, 'findUnique').mockResolvedValue(guestUser);
+    jest.spyOn(prisma.change_Request, 'findUnique').mockResolvedValue(changeBatmobile);
+
+    const res = await request(app).post('/create').send(createWorkPackagePayload);
+
+    expect(prisma.user.findUnique).toHaveBeenCalledTimes(1);
+    expect(res.statusCode).toBe(401);
+    expect(res.body.message).toBe('Access Denied');
+    afterEach(() => {
```

Comment:
the `afterEach` goes directly inside the describe (not inside tests) and it already exists in this file so you don't need to make a new one